graphjet-core/src/main/java/com/twitter/graphjet/algorithms/IdentityIDMask.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2017 Twitter. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twitter.graphjet.algorithms;
+
+public class IdentityIDMask implements IDMask {
```

Comment:
Right now it's only used for Moments, but I will keep it generic for any other entity that is not using masks to encode information into the IDs (for example, URLs, Users etc)